mwdb/model/api_key.py

```diff
@@ -28,14 +27,8 @@ def issuer_login(self):
         return self.issuer and self.issuer.login
 
     @staticmethod
-    def verify_token(token):
-        s = JSONWebSignatureSerializer(app_config.mwdb.secret_key)
-        try:
-            data = s.loads(token)
-        except SignatureExpired:
-            return None
-        except BadSignature:
-            return None
+    def _verify_token(token):
```

Comment:
I think these methods should not be preceded with `_` as they're part of interface.

This only have sense for methods that are used only by other methods within the same class (private methods).